autopush_rs/src/server/mod.rs

```diff
@@ -338,9 +342,17 @@ impl Server {
         } else {
             ServiceChangeTracker::new(Vec::new())
         };
+        let region = env::var("AWS_LOCAL_DYNAMODB").map(|endpoint| {
```

Comment:
nitpick: unwrap_or_else would be preferred here (don't care much because this is for tests)

Reply:
But it needs to construct the Region object with the value?